deployments/gpu-operator/values.yaml

```diff
@@ -294,6 +294,17 @@ devicePlugin:
     # MPS root path on the host
     root: "/run/nvidia/mps"
 
+draDriver:
+  enabled: true
+  repository: ghcr.io/nvidia
```

Comment:
should we mirror it to nvcr.io similar to other images and may be retag with semvar versioning

Reply:
Yes, this is a placeholder for now. We will update this once we publish the DRA driver image to nvcr.io